.github/workflows/build-and-publish-docs.yml

```diff
@@ -11,14 +11,20 @@ jobs:
   build-and-publish-docs:
     concurrency: ci-${{ github.ref }}
     runs-on: ubuntu-latest
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+#     TODO: Uncomment this check once running workflow manually is note needed anymore
+#    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
note: PR for restoring this check will be added right after we manage do build and publish them. Without it, we won't be able to manually run this workflow, because of previous run failure.